src/components/pages/RoleSignup/Applications/Styles/menteeApplication.css

```diff
@@ -27,21 +31,45 @@
   margin-bottom: 2vh;
 }
 
-@media only screen and (max-width: 600px) {
+@media only screen and (max-width: 575px) {
   .menteeApplication {
     padding: 0;
   }
+  .ant-form-large .ant-form-item-label > label {
+    height: 80px;
+}
 }
 
-::placeholder {
-  color: #737373 !important;
+.mentorTitle {
+  background-color: #003d71;
+  color: #ffffff !important;
+  text-align: center;
+  width: 100%;
+  border-top-right-radius: 12px;
+  border-top-left-radius: 12px;
+  padding: 2rem;
 }
 
-.menteeCriteria,
-.menteeHope,
-.menteeSubject,
-.menteeConvictions{
-  display: flex !important;
-  justify-content: center;
-  flex-direction: column !important;
+.main-col-container {
+  padding: 0rem;
+}
+.row-container {
+  display: flex;
+}
+.row-container1 {
+  display: flex;
+  gap: 1rem;
+}
+
+.form-group {
+  display: flex;
+}
+@media only screen and (max-width: 600px) {
+  .main-col-container {
+    padding: 1rem;
+  }
+  .form-group {
+    display: flex;
+    gap: 0rem;
+  }
 }
```

Comment:
Solid CSS work here making the containers flexible and dynamic, as well as adding the media query.